sky/backends/backend_utils.py

```diff
@@ -1616,6 +1616,8 @@ def get_head_ssh_port(
     max_attempts: int = 1,
 ) -> int:
     """Returns the ip of the head node."""
+    del handle  # Unused.
+    del use_cache  # Unused.
```

Comment:
I used del approach similar to `del max_attempts` for format.sh to pass. I hope it is ok.